tests/test_indexer.py

```diff
@@ -650,7 +652,8 @@ def test_scrub_index_data(self):
         with self.subTest("with schema version"):
             'Extra fields are removed.'
             manifest = read_bundle_manifest(self.blobstore, self.test_bucket, self.bundle_key)
-            index_data = create_index_data(self.blobstore, self.test_bucket, manifest)
+            index_data = create_index_data(self.blobstore, self.test_bucket, BundleFQID.from_key(self.bundle_key).uuid,
```

Comment:
To make this less verbose, I would pass the key to `create_index_data()` and derive the uuid in that method. It would also fit naturally because the preceding parameter is the bucket.

Reply:
Done.